source/opt/basic_block.h

```diff
@@ -64,6 +64,11 @@ class BasicBlock {
     return const_iterator(&insts_, insts_.cend());
   }
 
+  iterator tail() {
```

Comment:
I didn't want to call it back(), because back() returns reference, not iterator.